<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1258</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="importImgBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Chọn ảnh</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="imageUrl">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>731</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>841</width>
      <height>701</height>
     </rect>
    </property>
    <property name="text">
     <string>Ảnh</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="objectDetectionRadioBtn">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>50</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Đếm số lượng đối tượng</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="licensePlateRadioBtn">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>80</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Đọc biển số xe</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="executeFunction">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>120</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Thực hiện</string>
    </property>
   </widget>
   <widget class="QLabel" name="informationError">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>10</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="resultTableWidget">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>160</y>
      <width>361</width>
      <height>601</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1258</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
